kuma/search/fields.py

```diff
@@ -37,5 +37,4 @@ def to_representation(self, value):
         args = [getattr(value, arg) for arg in self.args]
         kwargs = {arg: getattr(value, arg) for arg in self.kwargs}
         locale = getattr(value, 'locale', settings.LANGUAGE_CODE)
-        path = reverse(self.url_name, locale=locale, args=args, kwargs=kwargs)
-        return '%s%s' % (settings.SITE_URL, path)
+        return reverse(self.url_name, locale=locale, args=args, kwargs=kwargs)
```

Comment:
The only time absolute URLs should ever be used is when you high up in the view code. This is a low level code and should *not* concern itself with how the URL should be presented.